src/Storefront/Resources/views/storefront/page/account/register/index.html.twig

```diff
@@ -1,9 +1,5 @@
 {% sw_extends '@Storefront/storefront/base.html.twig' %}
 
-{% block base_head %}
-    {% sw_include '@Storefront/storefront/page/account/register/meta.html.twig' %}
-{% endblock %}
-
```

Comment:
Are no meta tags needed on the register index page? 🤔

Reply:
I also stumbled upon this. The file was deprecated as it does not define any block. So just the base meta tags from the extended base file are used. The deprecation comment said that there is a specific snippet for some meta information on the register page.

Reply:
As far as I can tell, the snippet `"account.registerTitle` is never used now. It is not `account.registerMetaTitle`. Set by `\Shopware\Storefront\Page\Account\Login\AccountLoginPageLoader::setMetaInformation`

Reply:
@tobiasberge Exactly